Client/mods/deathmatch/logic/Config.h

```diff
@@ -52,17 +52,17 @@ class CVaPassNext
 #define va_pass(valist) CVaPassNext(valist).svapassnext
 /*** va_pass() (passing of ... variable length arguments ***/
 
-// Min and max number of characters in chat messages
+// Min and max number of characters in chat messages (the message itself, not including player nick)
```

Comment:
I wish there was a way to merge these client and server configs together in a shared config file, but maybe out of scope for this PR.

Reply:
Yeah it also needs to be made clearer what exactly the difference between max chat length, and max **chat echo** length is (among other things). 

I've cleared it up a little bit here hopefully, including fixing an oversight in `Packet_ChatEcho` where `MAX_OUTPUTCHATBOX_LENGTH` was being used instead of `MAX_CHATECHO_LENGTH`.